### DISCUSSION TITLE
How to Change the default capture screen on Windows 10 ?

### DISCUSSION THREAD

**Discussion by belcmax** [flameshot-org/flameshot#2122] [General]
Hello,

Could somebody tell me how to change the default Capture key (**_from Print Screen_**) on Windows 10 to any other key?
Thank you for your assistance.

![image](https://user-images.githubusercontent.com/95070039/143615322-78525ec0-9e3c-4b3d-a9d6-684bcda714e6.png)

Regards,
Bel C

**Comment by Fajle-Rayhan** [comment]
no-way to change
if you using other screenshot tool then this one then you face the problem like i did, and to use your preferred screenshot tool key properly,
you have to shut all screenshot tool then open your preferred one. i hope this will help your issues.

**Comment by ndnam7693** [comment]
Fix the key for capture screen isn't good.

**Comment by mmahmoudian** [comment]
Here is some help for Windows users:

https://flameshot.org/docs/guide/windows-help/
